### DISCUSSION TITLE
[灌水]（如果你害怕蜘蛛请不要打开）

### DISCUSSION THREAD

**Discussion by KELERSAMA** [share121/inter-knot#5452] [灌水]
![1727501960148](https://github.com/user-attachments/assets/c0e38501-20a6-447a-9e7a-d36528310568)

![IMG_20240928_131826](https://github.com/user-attachments/assets/90054cf1-6e3e-49f0-bbf5-7a2a366d5064)

2024.9.28
距离上次离家9.20
这只蜘蛛已经一个星期没动过了 就呆在窗帘上 甚至不怕人了
是在蜕皮吗
压迫感有点大

**Comment by SteveNotA** [comment]
![mmexport1727506229884](https://github.com/user-attachments/assets/0da650eb-dcef-4190-9cee-7c6687dcb2dd)

**Comment by fzdqz** [comment]
有啥好怕的，杀了得了
